monitor/views.py

```diff
@@ -89,11 +98,41 @@ def monitor_home(request):
              "sounds_analysis_failed_count": sounds_analysis_failed_count,
              "sounds_analysis_skipped_count": sounds_analysis_skipped_count,
              "gearman_status": gearman_status,
-             "sounds_in_moderators_queue_count": sounds_in_moderators_queue_count}
+             "sounds_in_moderators_queue_count": sounds_in_moderators_queue_count,
+    }
 
     return render(request, 'monitor/monitor.html', tvars)
 
 
+@cache_page(60 * 60 * 24)
+def stats_ajax(request):
+    last_week = datetime.datetime.now()-datetime.timedelta(weeks=1)
+
+    new_sounds = sounds.models.Sound.objects.filter(created__gt=last_week)\
+            .extra(select={'day': 'date(created)'}).values('day').order_by()\
+            .annotate(Count('id'))
+
+    new_users = User.objects.filter(date_joined__gt=last_week)\
+            .extra(select={'day': 'date(date_joined)'}).values('day')\
+            .order_by().annotate(Count('id'))
```

Comment:
Again this could be two lines, one with is_active=True, the other is_active=False. In this way we will quickly spot if there are problems with activation at some point.